src/runtime/StaticStrings.h

```diff
@@ -310,6 +310,7 @@ namespace Escargot {
     F(length)                     \
     F(let)                        \
     F(link)                       \
+    F(ListFormat)                 \
```

Comment:
(tiny fix) Word starts with upper case is located above according to the sorting rule.

Reply:
Okay!